src/envs/behavior.py

```diff
@@ -75,7 +81,11 @@ def __init__(self) -> None:
                     params_space=Box(parameter_limits[0], parameter_limits[1],
                                      (param_dim, )),
                     env=self.igibson_behavior_env,
-                    controller_fn=controller_fn,  # type: ignore
+                    planner_fn=planner_fn,
+                    # NOTE: type ignores below are necessary because these
+                    # functions take inputs of slightly different types.
```

Comment:
hmm, where does the difference arise? it looks to me like the policy and option model functions now both just take in plans, is that not the case?

Reply:
yes, but in the case of navigation, a plan is a list of tuples, and otherwise it is a list of lists. I think i can standardize on a list of lists to fix this though